app/react/src/server/build.js

```diff
@@ -80,6 +80,14 @@ webpack(config).run((err, stats) => {
   if (err) {
     logger.error('Failed to build the storybook');
     logger.error(err.message);
+  }
+  if (stats.hasErrors()) {
+    logger.error('Failed to build the storybook');
+    stats.toJson().errors.forEach(function (e) {
+      return logger.error(e);
+    });
+  }
+  if (err || stats.hasErrors()) {
```

Comment:
that would not print the stats errors if there is a "normal" error - is that deliberate?